src/components/organisms/Warsaw/Content.jsx

```diff
@@ -6,11 +6,18 @@ import imagePathWebp700 from './office_700.webp';
 import imagePathWebp1400 from './office_1400.webp';
 import imagePathJpg700 from './office_700.jpg';
 import imagePathJpg1400 from './office_1400.jpg';
+import Section from '../../molecules/Section';
```

Comment:
Move it upper. Leave imports of images at the end. Local component files should be always in the end.